airflow-core/src/airflow/ui/src/components/Graph/TaskNode.tsx

```diff
@@ -106,7 +104,6 @@ export const TaskNode = ({
                 <StateBadge fontSize="xs" state={taskInstance.state}>
                   {taskInstance.state}
                 </StateBadge>
-                {taskInstance.try_number > 1 ? <CgRedo /> : undefined}
```

Comment:
this also is information not on the object. does not seem of great consequence though.

Reply:
ok thank you